### ISSUE TITLE
Token can't be used yet - check the server's date/time matches the client

### ISSUE THREAD

**Issue by lowingly** [TuranicTeam/Altay#54]
When I enter the service:
11:29:38 INFO > LolitaQAQ[/192.168.0.12:59591] logged out due to Invalid session. Reason: Token can't be used yet - check the server's date/time matches the client.

I was able to play normal on the server yesterday.
But today's server has a problem.
I uninstalled all the plugins and could not solve them.
I reopened a brand new server and couldn't solve it.
I tried to modify my computer time and mobile phone time and still can't solve it.
Hope this is not a stupid question. I hope to get a detailed answer. Thank you

Altay 1.2 「TuranicPro」 implementing API version 3.0.1 
for Minecraft: PE v1.2.10 (protocol version 201)
Windows 8

**Comment by lowingly** [comment]
Add a possible time issue. I noticed that my computer time is different from the server time. My server time is one hour slower than my computer time. I don't know if this is related to the upcoming "summer time."

**Comment by Enes5519** [comment]
"xbox-auth" set to  "false" in server.properties

**Comment by lowingly** [comment]
I changed it to false. But the problem is not resolved....

**Comment by ErkamKahriman** [comment]
It has been to do with our system time but i should be fixed it you set xbox-auth to `false`

**Comment by ErkamKahriman** [comment]
Or correct our system time

**Comment by lowingly** [comment]
What else can I do to try?
Changing "xbox-auth" to "false" did not solve the problem.
My computer system time I modified many times.
I had successfully entered many times before..but now I can't get into it.
I also tried using Turinic (1.1.x) to create the server and everything worked. I can enter the Turanic server, but Altay can't...
I don't know what happened..It seems that only I have this problem. Maybe it will be normal if I try to open the service in a few days......T_T
